client/src/components/AddNewProfile.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import { Form } from "semantic-ui-react";
+import { getProfiles } from "../api/profiles";
+
+const AddNewProfile = () => {
+  const [profileData, setProfileData] = useState({
+    firstname: "",
+    lastname: "",
+    dob: "",
+    address: "",
+    email: "",
+    phone: "",
+    gender: "",
+    groups: "",
+    support_type: ""
+  });
+
+  const [profileCreated, setProfileCreated] = useState(null);
+
+  const handleChange = event => {
+    const updatedProfileData = {
+      ...profileData,
+      [event.target.name]: event.target.value
+    };
+
+    setProfileData(updatedProfileData);
+  };
+
+  const createProfile = event => {
+    getProfiles(profileData).then(isSuccessful => {
+      console.log(isSuccessful);
```

Comment:
It's not good practise to have `console.log` statements in frontend production code — let's remove it